pkg/datastore/headerscontroller.go

```diff
@@ -0,0 +1,282 @@
+package datastore
+
+import (
+	"sort"
+	"sync"
+	"sync/atomic"
+	"time"
+
+	"github.com/blocknative/dreamboat/pkg/structs"
+)
+
+type HeaderController struct {
+	content map[uint64]*HNTs
```

Comment:
Could we replace `content` by `headers`  in this and the other structs? In order to make what the content is clear

Reply:
done